frontend/src/views/edit-project/RepresentationArea.tsx

```diff
@@ -36,7 +36,10 @@ export const RepresentationArea = ({
 }) => {
   let content;
   if (!displayedRepresentation) {
-    content = <OnboardArea selection={selection} setSelection={setSelection} />;
+    if (window.performance) {
+      // If the user reloads a diagram page, it should load the OnboardArea
+      content = <OnboardArea selection={selection} setSelection={setSelection} />;
+    }
```

Comment:
Leftover from previous version? This API should not be used (and it should not be necessary to fix the issue).

Reply:
Indeed I forgot to remove this "if"